### ISSUE TITLE
white tab text on dark tab background is difficult to read

### ISSUE THREAD

**Issue by bradleyrichter** [brave/browser-laptop#8115]
The font weight may need to change for the white tab text label case.

**Comment by bradleyrichter** [comment]
@cezaraugusto I found a solution:

![image](https://cloud.githubusercontent.com/assets/13509546/24818490/a1ec120c-1b95-11e7-918a-e8ba05dee3bb.png)

So, the weight should be 600 only for the selected tab which is displaying white text based on the BG color threshold.
